extension-api/package.json

```diff
@@ -10,11 +10,12 @@
   },
   "author": "David Woodruff",
   "license": "Apache-2.0",
-  "dependencies": {
+  "peerDependencies": {
     "aws-sdk": "^2.188.0"
   },
   "devDependencies": {
     "@types/node": "^9.4.0",
+    "aws-sdk": "^2.188.0",
```

Comment:
...and to make Typescript happy, we have to install it as a dev dependency too 🤷‍♂️